src/screens/MainScreen/styles.js

```diff
@@ -19,18 +73,30 @@ export const Title = styled.div`
   line-height: ${themeGet("fonts.title.lineHeight")};
   letter-spacing: ${themeGet("fonts.title.letterSpacing")};
   color: ${themeGet("colors.textTitle")};
-  padding: 5vh;
+  padding: 5vh 0;
 `;
 
+export const WhiteBgTitle = styled.div`
+  text-align: center;
+  font-size: ${themeGet("fonts.title.size")};
+  font-family: ${themeGet("fonts.title.family")};
+  font-weight: ${themeGet("fonts.title.weight")};
+  line-height: ${themeGet("fonts.title.lineHeight")};
+  letter-spacing: ${themeGet("fonts.title.letterSpacing")};
+  color: ${themeGet("colors.textTitle")};
+  padding: 5vh 0;
+  width: 100%;
+  background-color: ${themeGet("colors.white")};
+`
+
 export const Subtitle = styled.div`
   text-align: center;
   font-size: ${themeGet("fonts.subtitle.size")};
-  font-family: ${themeGet("fonts.subtitle.family")};
+  font-family: Gravity-Light;
```

Comment:
Best to put this in the relevant place in the theme.js rather than hardcoding it

Reply:
done.